test/uk/gov/hmrc/helptosave/services/UserCapServiceSpec.scala

```diff
@@ -149,15 +153,15 @@ class UserCapServiceSpec extends TestSupport with ServicesConfig {
       val userCapService = new UserCapServiceImpl(userCapStore, configOverride)
 
       "allow account creation as long as totalCap is not reached" in {
-        val userCap = UserCap(yesterday, 113, 100)
+        val userCap = UserCap(yesterday, dailyLimit, totalLimit - 1)
         mockUserCapStoreGetOne(Some(userCap))
 
         result(userCapService.isAccountCreateAllowed()) shouldBe UserCapResponse()
       }
 
       "not allow account creation if totalCap is reached " in {
 
-        val userCap = UserCap(today, 9, 10000)
+        val userCap = UserCap(today, dailyLimit, totalLimit)
```

Comment:
Should this be the following to test just the total cap?
val userCap = UserCap(today, dailyLimit - 1, totalLimit)